src/main/kotlin/no/nav/syfo/client/saf/SafDokumentClient.kt

```diff
@@ -35,7 +34,7 @@ class SafDokumentClient constructor(
             httpClient.get<HttpStatement>("$url/hentdokument/$journalpostId/$dokumentInfoId/ORIGINAL") {
                 accept(ContentType.Application.Xml)
                 header("Authorization", "Bearer ${stsClient.getToken()}")
-                header("Nav-Callid", MDCOperations.putToMDC(MDCOperations.MDC_CALL_ID, UUID.randomUUID().toString()))
+                header("Nav-Callid", MdcUtils.getCallId())
```

Comment:
`putToMDC` returnerte ingenting, så jeg antok at dette måtte være en feil og at man heller ønsket å hente call-ID-en, som er gjort for de andre http-kallene.